gensrc/thrift/StarRocksExternalService.thrift

```diff
@@ -114,7 +114,7 @@ struct TScanCloseResult {
 }
 
 // scan service expose ability of scanning data ability to other compute system
-service TDorisExternalService {
+service TStarRocksExternalService {
```

Comment:
I prefer TStarrocks, becuase starrocks is one word. TStarRocksExternalService make starrocks like two words.